src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TestProgressState.cs

```diff
@@ -45,7 +45,7 @@ public TestProgressState(long id, string assembly, string? targetFramework, stri
 
     public long Version { get; internal set; }
 
-    public List<(string? DisplayName, string? UID)> DiscoveredTests { get; internal set; } = [];
+    public List<(string DisplayName, string UID)> DiscoveredTests { get; internal set; } = [];
```

Comment:
@Youssef1313 i dont think `UID` is used anywhere. should it be?

Reply:
Merging as-is, let's do a follow-up